website/docs/registry/providers/publishing.html.md

```diff
@@ -14,7 +14,25 @@ description: |-
 
 Providers published to the Terraform Registry are written and built in the same way as other Terraform Providers. For guidance on how to write a provider, see [Writing Custom Providers](/docs/extend/writing-custom-providers.html).
 
-The provider repository on GitHub must match the pattern `terraform-provider-{NAME}`, and the repository must be public.
+The provider repository on GitHub must match the pattern `terraform-provider-{NAME}`, and the repository must be public.  
```

Comment:
@paultyng @justincampbell updated with separate header, and made it clear this requirement applies to partners only